src/app/components/phone-field/index.html

```diff
@@ -1,10 +1,11 @@
 <sky-tabbed-demo-page
-  pageTitle="Phone field"
+  pageTitle="Phone Field"
+  navTitle="Phone field"
 >
 
   <sky-demo-page-summary>
     <p>
-      The phone field module creates a button and text input to validate international phone numbers. The phone field button indicates the selected country and enables users to select different countries. Users enter phone numbers in the text input, and then the phone field input directive validates the numbers against standards for the selected country. For validation, phone numbers are driven through the <sky-code>ngModel</sky-code> attribute that you specify on an <sky-code>input</sky-code> element or on a <sky-code>FormControl</sky-code> in a reactive form.
+      The phone field module creates a button, search input, and text input for entering and validating international phone numbers. The country selector button displays a flag icon for the selected country, and users enter phone numbers in the phone number input. The phone field input directive then validates the phone numbers based on the format for the selected country. To change the selected country, users select the country selector button to expose the country search input.
     </p>
```

Comment:
Can we shorten the page summary text? It seems like we can get away with just the first sentence you propose since the rest of the content is addressed in the docs.